src/components/Tabs.react.js

```diff
@@ -121,6 +121,96 @@ EnhancedTab.defaultProps = {
     },
 };
 
+const NextTab = ({
```

Comment:
This file is becoming very big. Consider breaking up the `Tabs` component file into multiple files. One way to do this would be to rename `src/components/Tabs.react.js` to `src/components/Tabs.react/index.js` (existing imports will recognize `<folder>/index.js` as the same as `<folder>.js` and won't require changes) and moving `EnhancedTabs` and `NextTab` into files of their own, referenced here.